### PR TITLE
1313e dev

### PR DESCRIPTION
**Pull request by 1313e**
Let's open another one of these :)

### CODE REVIEW

**Comment by 1313e** [comment]
Alright, I think I am done.
The remaining tests will require more work, and I don't know there what you want with them.

File: doc/source/conf.py [hunk 1]
```diff
@@ -10,30 +10,40 @@
...
+
+# Determine the names of all requirements
+req_names = set(map(lambda x: x.name, parsed_reqs))
+docs_req_names = set(map(lambda x: x.name, parsed_docs_reqs))
+
+# Take the difference between req_names and docs_req_names
+names = req_names.difference(docs_req_names)
+
+# Create mock modules for all modules IMAGINE needs but the docs do not
+for mod_name in names:
```
Comment (luizfelippesr) [inline doc/source/conf.py:39]: Clever!

File: imagine/tests/test_observabledict.py [hunk 2]
```diff
@@ -140,7 +137,7 @@ def test_covdict_apply_mask(self):
         covdict.append(('test', 'nan', str(2*mpisize), 'nan'), cov, True)
         covdict.apply_mask(mskdict)
         pix_num = msk.sum()
-        self.assertTrue(('test', 'nan', str(pix_num), 'nan') in covdict.keys())
+        assert ('test', 'nan', str(pix_num), 'nan') in covdict.keys()
 
 if __name__ == '__main__':
     unittest.main()
```
Comment (luizfelippesr) [inline imagine/tests/test_observabledict.py:143]: Will this still work?

Comment (1313e) [reply imagine/tests/test_observabledict.py:143]: Do you think I would push it if it didn't work properly?

Comment (luizfelippesr) [reply imagine/tests/test_observabledict.py:143]: I was referring specifically to  the last lines:
```python
if __name__ == '__main__':
    unittest.main()
```
as thought `unittest` was removed from the module

Comment (1313e) [reply imagine/tests/test_observabledict.py:143]: Uhm, that's weird.
Those lines are not supposed to be there.

Comment (luizfelippesr) [reply imagine/tests/test_observabledict.py]: That's what code review is for!   : - )

Comment (1313e) [reply imagine/tests/test_observabledict.py]: Well, it would still work, until you execute the file instead of importing it (which is what `pytest` does).

**Pull request merged**